charts/datarepo-api/templates/api-deployment.yaml

```diff
@@ -13,8 +13,11 @@ spec:
   template:
     metadata:
       labels:
+{{- include "datarepo-api.labels" . | nindent 8 }}
         app: {{ include "datarepo-api.fullname" . }}
         component: {{ include "datarepo-api.fullname" . }}
+      annotations:
```

Comment:
See above.

Reply:
ability to add any annotation into a deployment
ie:
```
      annotations:
        prometheus.io/scrape: 'true'
        prometheus.io/port: '9102'
```

Reply:
add comments